eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -481,7 +481,7 @@ public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cau
     }
 
     @Sharable
-    private class HttpConnectionHandler extends ChannelDuplexHandler {
+    protected class HttpConnectionHandler extends ChannelDuplexHandler {
 
```

Comment:
Is there any way to keep it private here?

Reply:
EventMeshAdminServer can not use If this HttpConnectionHandler  keep private .